UPGRADE-0.x.md

```diff
@@ -1,6 +1,8 @@
 UPGRADE 0.x
 ===========
 
+### Add support for sonata-project/datagrid-bundle:^3.2
```

Comment:
Bumping a major constraint is not a synonym of adding support.
Adding support is the case when you add support over the existing compatibility, IE `^2.3|^3.2`. But you are dropping support for `^2.3` here.

Reply:
ahh sry, i forget to update it

Reply:
Almost there.